.github/workflows/build-and-release.yml

```
@@ -20,17 +20,16 @@ jobs:
-      - name: Install depcheck
-        run: npm install -g depcheck
-
-      - name: Check for unused dependencies (ignore devDependencies)
-        shell: bash
-        run: |
-          IGNORES=$(node -p "Object.keys(require('./package.json').devDependencies).join(',')")
-          depcheck --ignores="$IGNORES,electron,webpack,webpack-cli,@eslint/js" || (echo "Unused dependencies found" && exit 1)
-
-      - name: Run tests
-        run: npm run test:ci
+      - parallel:
```

actionlint (1.7.12)

[error] step must run script with "run" section or run action with "uses" section

(syntax-check)